bgpd/bgp_evpn.c

```diff
@@ -4333,6 +4333,57 @@ static void update_autort_vni(struct hash_bucket *bucket, struct bgp *bgp)
 	}
 }
 
+/*
+ * Handle autort change for L3VNI.
+ */
+static void update_autort_l3vni(struct bgp *bgp)
+{
+	if ((CHECK_FLAG(bgp->vrf_flags, BGP_VRF_IMPORT_RT_CFGD))
+	    && (!CHECK_FLAG(bgp->vrf_flags, BGP_VRF_EXPORT_RT_CFGD)))
```

Comment:
This is wrong. You should return only if the RTs are configured.

Reply:
thanks for pointing this out. It was a bad review from my side when I made some changes.